presto-main/src/test/java/com/facebook/presto/sql/planner/optimizations/TestLocalProperties.java

```diff
@@ -703,6 +713,34 @@ public void testDifferentSortOrders()
                 Optional.of(sorted("a", SortOrder.ASC_NULLS_LAST)));
     }
 
+    @Test
+    public void testJsonSerialization()
+            throws Exception
+    {
+        ObjectMapper mapper = new ObjectMapperProvider().get()
+                .registerModule(new SimpleModule()
+                        .addDeserializer(ColumnHandle.class, new JsonDeserializer<ColumnHandle>()
```

Comment:
This is duplicated in `TestDupleDomain`. Create a `TestingColumnHandleDeserializer`?